src/Symfony/Component/Security/Http/Authenticator/AccessToken/AccessTokenExtractorInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Authenticator\AccessToken;
```

Comment:
I would use the same organization as e.g. remember me and login link authenticators:

* Add the main authenticator class to `Symfony\Component\Security\Http\Authenticator`.
* Create a subnamespace `Symfony\Component\Security\Http\AccessToken` for the extractors/other classes

Reply:
OK. Changes made in the last commits